### ISSUE TITLE
User and space avatar are not accessible.

### ISSUE THREAD

**Issue by akhanfir** [Meeds-io/meeds#1356]
**Steps to reproduce**

1. Create a space

2. Create and activity

3. Refresh stream

**Current behaviour**

- the a tag of avatar have not a discernible name

- the img tag for avatar have a role="presentation" which is not the correct one

- Space and User avatar are concerned

**Expected behavior**

*For User Avatar

- The a tag must have a dicernible name, set with aria-label attribute. The value must be what the link do : "Open {userFullname} profile", and it must be i18n, in french for example : "Ouvrir le profil de {userFullname}". 

- The image tag must have not the role="presentation", and have a alt attribute with value = "{userFullname}'s avatar", and it must be i18n. In french for example "Avatar de {userFullname}"

*For Space Avatars

- The a tag must have a dicernible name, set with aria-label attribute. The value must be what the link do : "Open space {spaceDisplayName}", and it must be i18n, in french for example : "Ouvrir l'espace {spaceDisplayName}". 

- The image tag must have not the role="presentation", and have a alt attribute with value = "{ Space {spaceDisplayName} avatar", and it must be i18n. In french for example "Avatar de l'espace { {spaceDisplayName}".
